src/components/application_manager/src/message_helper.cc

```diff
@@ -1344,6 +1344,13 @@ smart_objects::SmartObjectList MessageHelper::CreateAddSubMenuRequestToHMI(
     msg_params[strings::menu_id] = i->first;
     msg_params[strings::menu_params][strings::position] =
         (*i->second)[strings::position];
+
```

Comment:
@VVeremjova  As far as I understand you need to add supporting saving this new parameter to resumption DB. Please clarify this with Melnyk T.